api/auth.js

```diff
@@ -8,116 +8,116 @@ const JWT_SECRET = process.env.JWT_SECRET;
 
 // create token
 function createToken(id) {
-    return jwt.sign({ id }, JWT_SECRET, { expiresIn: "1d" });
+  return jwt.sign({ id }, JWT_SECRET, { expiresIn: "1d" });
 }
 
 // Email validation regex
 const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
 
 // token checking middleware
 router.use(async (req, res, next) => {
-    const authHeader = req.headers.authorization;
```

Comment:
I did changes here but then reverted them cause bertha reminded me of the easier way. So everything is the same here.